CHANGELOG.md

```diff
@@ -45,9 +63,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - The async_operation_image property of cumulus module should be updated to pull
   the ECR image for cumuluss/async-operation:47
 
+## [v16.0.0] 2023-05-09
+
 ### MIGRATION notes
 
-#### RDS Phase 3
+#### PI release version
+
+When updating directly to v16 from prior releases older that V15, please make sure to
+read through all prior release notes.
+
+Notable migration concerns since the last PI release version (11.1.x):
```

Comment:
Is 11.1.x correct?

Reply:
Confirmed that 11 was the prior supported version and 11-16 and 15-16 were tested.